yt_dlp/extractor/youtube.py

```diff
@@ -2227,7 +2235,8 @@ class YoutubeIE(YoutubeBaseInfoExtractor):
                 'thumbnail': 'https://i.ytimg.com/vi_webp/vv8qTUWmulI/maxresdefault.webp',
                 'uploader_url': 'http://www.youtube.com/user/BlastfromthePast',
                 'channel_url': 'https://www.youtube.com/channel/UCzIZ8HrzDgc-pNQDUG6avBA',
-            }
+            },
+            'skip': 'This video has been removed by the uploader',
```

Comment:
In my opinion this test should be replaced, rather than being disabled. I have no idea how to find Youtube stories though

Reply:
Yeah, it is an issue. I find they last more than 7 days  before becoming unavailable - more like a month of so.

Maybe it would be better to add a test in the story IE that extracts all the way. Use a channel like LTT that usually always has at least one story.

Or could just skip the test and find out it doesn't work when someone reports in 5 years 😉